silhouette/app/com/mohiva/play/silhouette/impl/providers/SocialProvider.scala

```diff
@@ -165,6 +165,14 @@ trait SocialProfile {
   def loginInfo: LoginInfo
 }
 
+trait IdentifiedSocialProfile extends SocialProfile {
```

Comment:
Why do you have implemented the `IdentifiedSocialProfile` trait? This isn't necessary and it forces every social profile to implement the fields and it breaks all the application which doesn't use the `CommonSocialProfileBuilder`.